### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by Tamunokuro**
- Add linter files in root directory

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @Tamunokuro, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no professional documentation, kindly consider using [the README template provided by microverse](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <ul className="navbar">
+        <li className="nav-items"><Link to="/">BOOKS</Link></li>
+        <li className="nav-items"><Link to="/categories">CATEGORIES</Link></li>
+      </ul>
+      <Routes>
+        <Route path="/" element={<Book />} />
+        <Route path="/categories" element={<Categories />} />
+        <Route />
+        <Route />
+      </Routes>
```
Comment (Ismailco) [inline src/components/Navbar.js]: - [ ] Kindly consider adding this part to the `index.js` or `App.js`, so your component is divided properly. the `Navbar` should include only the app navigation

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button, Container } from 'react-bootstrap';
+import InputBook from './InputBook';
+
+const Book = () => (
```
Comment (Ismailco) [inline src/components/Book.js]: - [ ] You should have a separate component `BookList` that renders a list of books using the `Book` component and renders a form by using the `CreateNewBook` component (InputBook in your case), kindly consider creating this component so it's easy to add and display new books

Comment (Ismailco) [inline src/components/Book.js]: - [ ] The `Book` component should take the book data as a **prop** and renders the title and the author, with a button to remove the book, kindly consider implementing this so your component can take as many books as needed not just hard-coded books

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Tamunokuro, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand:  @Tamunokuro, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

It seems you have submitted the wrong project for review. Kindly double-check before submitting once more. The link you submitted has already been reviewed. Once more, always make sure you have a good title and a good PR summary. The title of your project is **Add linters #1** This title is not descriptive.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**